QuasarHelios2015/src/org/team1540/quasarhelios/Clamp.java

```diff
@@ -23,16 +30,16 @@
 
     public static void setup() {
 
-//        FloatInputPoll encoder = Igneous.makeEncoder(10, 11, false);
-        FloatInputPoll encoder = FloatMixing.always(0.0f);
-        //FloatOutput speedControl = Igneous.makeTalonMotor(2, Igneous.MOTOR_REVERSE, 0.1f);
+        FloatInputPoll encoder = Igneous.makeEncoder(10, 11, false);
         ExtendedMotor clampCAN = Igneous.makeCANTalon(1);
-        FloatOutput speedControl;
+        FloatOutput f = new FloatStatus();
         try {
-            speedControl = clampCAN.asMode(ExtendedMotor.OutputControlMode.VOLTAGE_FRACTIONAL);
+            f = clampCAN.asMode(ExtendedMotor.OutputControlMode.VOLTAGE_FRACTIONAL);
         } catch (ExtendedMotorFailureException e) {
-            throw new RuntimeException(e);
+            Logger.log(LogLevel.SEVERE, "Exception throw when creating clamp motor: ", e);
```

Comment:
Don't need the colon and space, and `thrown` would probably be more accurate. Also you can just use `Logger.severe`.